test/catalyst/catalyst.test.js

```diff
@@ -11,13 +11,15 @@ function testCatalyst(catalystName) {
   const erc20Tests = generateERC20Tests(
     async () => {
       const others = await getUnnamedAccounts();
-      const {deployer} = await getNamedAccounts();
+      const {catalystOwner} = await getNamedAccounts();
```

Comment:
here instead of getting `catalystOnwer` from `getNamedAccounts` you can get it from others